src/raudio.c

```diff
@@ -1726,7 +1729,12 @@ void UpdateMusicStream(Music music)
     unsigned int subBufferSizeInFrames = music.stream.buffer->sizeInFrames/2;
 
     // NOTE: Using dynamic allocation because it could require more than 16KB
-    void *pcm = RL_CALLOC(subBufferSizeInFrames*music.stream.channels*music.stream.sampleSize/8, 1);
+    size_t pcmSize = subBufferSizeInFrames * music.stream.channels * music.stream.sampleSize / 8;
+    if (AUDIO.System.pcmCapacity < pcmSize) {
+        RL_FREE(AUDIO.System.pcm);
+        AUDIO.System.pcm = RL_CALLOC(1, pcmSize);
```

Comment:
You should obviously just use REALLOC here instead, but the compiler was whining about possible memory leaks due to not checking return value of realloc before overwriting the pointer. -_-